scripts/ci.js

```diff
@@ -43,21 +45,27 @@ const isHeadCommitOfPR = async () => {
 };
 
 const stackReadyForDeploy = async () => {
-  const params = {
-    StackName: `sls-${name}-${tier}-${stage}`
-  };
+  const stack = `sls-${name}-${tier}-${stage}`;
+  let stackMsg = 'is ready';
 
-  // eslint-disable-next-line no-console
-  console.log('\nChecking if CloudFormation stack is ready...');
+  log('\nChecking if CloudFormation stack is ready...');
 
   await Promise.race(
     [('stackCreateComplete', 'stackUpdateComplete')].map(event =>
-      cloudformation.waitFor(event, params).promise()
+      cloudformation.waitFor(event, { StackName: stack }).promise()
     )
-  );
+  ).catch(err => {
+    // Allow non-existent CF stack (like on initial PR creation).
+    const originalError = err.originalError || {};
+    if (originalError.message === `Stack with id ${stack} does not exist`) {
+      stackMsg = 'does not exist (continuing)';
+      return;
+    }
```

Comment:
Bugfix for `'Stack with id sls-badges-nonprod-pr49 does not exist'` on initial PR creation.

This does introduce a potential slight race condition when sls CF stack doesn't exist but it's better than a failure and should be a small window.

Outputs on initial:

```
Checking if CloudFormation stack is ready...
CloudFormation stack does not exist (continuing)
```

Outputs on subsequent:

```
Checking if CloudFormation stack is ready...
CloudFormation stack is ready
```

/cc @kevinmstephens